pom.xml

```diff
@@ -28,8 +28,20 @@
 
 	<dependencyManagement>
 		<dependencies>
-				<dependency><groupId>org.openmrs.contrib</groupId><artifactId>testutils</artifactId><version>${openmrsTestutilsVersion}</version><scope>test</scope></dependency>
-
+			<dependency><groupId>org.openmrs.contrib</groupId><artifactId>testutils</artifactId><version>${openmrsTestutilsVersion}</version><scope>test</scope></dependency>
+			<dependency>
+				<groupId>org.openmrs.module</groupId>
+				<artifactId>webservices.rest-omod</artifactId>
+				<version>${webServicesRestVersion}</version>
+				<scope>provided</scope>
+			</dependency>
+			<dependency>
+				<groupId>org.openmrs.module</groupId>
+				<artifactId>webservices.rest-omod-common</artifactId>
+				<version>${webServicesRestVersion}</version>
+				<classifier>tests</classifier>
```

Comment:
Then there is something wrong you setup and needs to be fixed, classifier has nothing to do with the jar being test or not, classifier is just a logical phrase you include in the artifact id  for naming purposes e.g RC2, europa, ganymede, helios etc, they mean nothing to the code